samples/tutorials/mnist/01_Lightweight_Python_Components.ipynb

```diff
@@ -247,7 +243,8 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "divmod_op = comp.func_to_container_op(my_divmod)"
+    "divmod_op = comp.func_to_container_op(func=my_divmod, \n",
+    "                                      base_image=\"tensorflow/tensorflow:1.15.0-py3\")"
```

Comment:
Does it really need the tensorflow image?

Reply:
Not really. As long as having  base container image with python 3.5+ and Numpy installed I think. Previously, the tensorflow IO is used to dump the meta data to a file. It still serve the purpose though.